src/snakemake/workflow.py

```
@@ -266,6 +266,47 @@ def async_run(self, coro):
+        from datetime import datetime
+        from snakemake.common import __version__
+
+        return {
+            "datetime": datetime.now().strftime("%Y-%m-%d %H:%M:%S"),
+            "snakemake_version": __version__,
+            "platform": platform.platform(),
+            "host": platform.node(),
+            "user": getpass.getuser(),
+            "conda_version": (
+                subprocess.run(
+                    ["conda", "--version"],
```

Ruff (0.15.5)

[error] Starting a process with a partial executable path

(S607)